themes/bootstrap3/templates/layout/layout.phtml

```diff
@@ -107,6 +107,12 @@
     ?>
     <?=$this->cookieConsent()->render()?>
     <?=$this->headScript() ?>
+    <?php if(isset($lightboxParent)): ?>
```

Comment:
Minor style -- we use `if (` instead of `if(`, and I don't think you need the opening whitespace inside the `<script>` tags.

Reply:
fixed